storage/kv_store_leveldb_num_keys.py

```diff
@@ -1,28 +1,29 @@
+import os
+import shutil
 from typing import Iterable, Tuple
 
-import shutil
-from state.kv.kv_store import KeyValueStorage
 from state.util.utils import removeLockFiles
+from storage.kv_store import KeyValueStorage
 
 try:
     import leveldb
 except ImportError:
     print('Cannot import leveldb, please install')
 
 
-class KeyValueStorageLeveldb(KeyValueStorage):
-    def __init__(self, dbPath, open=True):
+class KeyValueStorageLeveldbNumKeysSupport(KeyValueStorage):
```

Comment:
Looks like we don't use it, some renaming artifact. I will remove the class. Thanks for finding.